types/algoliasearch-helper/index.d.ts

```diff
@@ -0,0 +1,891 @@
+// Type definitions for algoliasearch-helper 2.26
+// Project: https://community.algolia.com/algoliasearch-helper-js/
+// Definitions by: Gordon Burgett <https://github.com/gburgett>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import { Client } from 'algoliasearch';
+import { EventEmitter } from 'events';
+
+/**
+ * The algoliasearchHelper module is the function that will let its
+ * contains everything needed to use the Algoliasearch
+ * Helper. It is a also a function that instanciate the helper.
+ * To use the helper, you also need the Algolia JS client v3.
+ * @param client an AlgoliaSearch client
+ * @param index the name of the index to query
+ * @param opts
+ */
+declare function algoliasearchHelper(client: Client, index: string, opts: Partial<algoliasearchHelper.SearchParameters>): algoliasearchHelper.AlgoliaSearchHelper;
```

Comment:
```suggestion
declare function algoliasearchHelper(client: Client, index: string, opts: algoliasearchHelper.SearchParameters): algoliasearchHelper.AlgoliaSearchHelper;
```

This doesn't need to be made `Partial`, since every parameter in the `algoliasearch` definitions is already declared as optional (or not)